### PR TITLE
Feature/foodnetwork

### PR DESCRIPTION
**Pull request by WagglyDessert**
____ Wrote Tests ____ Implemented ____ Reviewed

## Neccesary checkmarks:

    [x] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: added foodnetwork webscraping
    
## Requested feedback: 
    
    I'd like feedback on...  webscraping to see if it is sending out the right format

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: app/controllers/api/v1/searches_controller.rb [hunk 1]
```diff
@@ -34,6 +34,10 @@ def scrape_from_sites(website)
       facade = AllRecipesScrapeFacade.new
       recipe_results = facade.recipe_details(website)
       render json: WebScrapeSerializer.recipes(recipe_results), status: 200
+    elsif search_term.include?('https://www.foodnetwork.com')
```
Comment (skim1027) [inline app/controllers/api/v1/searches_controller.rb:37]: do you think it's better to leave https://www. out to be consistent with the other elsif statement?

**Review by skim1027 — COMMENTED** [review]
I have a comment on the controller

File: app/controllers/api/v1/searches_controller.rb (hunk 1 above)
Comment (WagglyDessert) [reply app/controllers/api/v1/searches_controller.rb:37]: yes let me change that

**Review by skim1027 — APPROVED** [review]
thank you for making the update!

**Pull request merged**